### ISSUE TITLE
Bug in raising division instructions.

### ISSUE THREAD

**Issue by deXetrous** [Microsoft/llvm-mctoll#15]
I have been using llvm-mctoll for my project which involves raising of binaries to IR. I have been getting an error when i try to raise binaries which are generated for a C code involving division operation on a variable. It works fine if both the operands are constants.
As an example: The following code snippet works fine.
```
int main() {
   int a = 4/2;
   return 0;
}
```
But, there seems to be an issue with variable operands.
As an example:
```
int main() {
  int a = 10;
  a = a/2;
  return 0;
}
```
While raising the binary, following error occurs:
```
*** Generic instruction not raised :   IDIV32r $ecx, <0x557f28d346e8>, implicit-def $eax, implicit-def $edx, implicit-def $eflags, implicit $eax, implicit $edx
```
Please let me know if this is not implemented or there is an issue from my side.

**Comment by bharadwajy** [comment]
Thanks for trying the tool and filing the bug report.

Support to raise several x64 instructions still needs to be added - division instructions are a subset of such. We are actively working towards adding support to missing instructions.

If you have the cycles, we welcome any help you can offer.

Thanks!

**Comment by bharadwajy** [comment]
Fix pushed (commit bca73b362c78e8bc44fafa01364e700d25378b82)
